stacks/kafka-flink/README.Unix.rst

```diff
@@ -13,6 +14,22 @@ This document will outline how to run the tutorial on Linux, macOS and WSL2.
 Setup
 *****
 
+Prerequisites
+=============
+
+In order to exercise the command walkthrough successfully, please make sure
+those programs are installed/available on your machine / in your environment.
+
+- bash
+- cat
+- curl
+- docker
+- git
+- head
+- jq
+- tar
+- wget
```

Comment:
not important, but we could replace wget with curl where needed to download a file

Reply:
Tha's true. I will consider it for a future iteration.